### PR TITLE
Purchase summary in sidebar

### PR DESCRIPTION
**Pull request by alex-deaconu**
## Description
Purchase summary in sidebar

[stage-2]

## Motivation and Context
Billing Friction epic

## How Has This Been Tested?
Tested changes locally; ensured unit & e2e tests pass.

## Release Plan:
- As the Submitter, upon requesting review of this pull request, I confirm that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed. 
- As the Reviewer, upon approving the changes in this PR, I confirm I have reviewed and I agree that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed

#### Release Checklist Items Skipped?
No

### CODE REVIEW

**Comment by alex-deaconu** [comment]
@ezequielc please review. Thanks!

File: web/partials/purchase/checkout-payment-methods.html [hunk 1]
```diff
@@ -1,202 +1,191 @@
...
-      <!-- Alpha Release - Select New Card by default -->
-      <div class="row" ng-if="false">
-        <div class="col-md-12">
-          <div class="form-group">
-            <label for="credit-card-select" class="hidden">Add New Credit Card</label>
-            <select id="credit-card-select" class="form-control selectpicker" ng-model="paymentMethods.selectedCard" ng-options="c as getCardDescription(c) for c in paymentMethods.existingCreditCards track by c.id">
-              <option value="">Add New Credit Card</option>
-            </select>
-          </div>
+<div class="purchase-sideways-panels">
```
Comment (alex-deaconu) [inline web/partials/purchase/checkout-payment-methods.html:2]: @ezequielc it helps to review this file without whitespace. Thanks.

File: web/partials/purchase/checkout-purchase-summary.html [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+    {{selectedCompany.id}}
+  </p>
+
+  <div class="border-bottom py-4 mb-4">
+    <p class="mb-0">
+      <span class="font-weight-bold">{{purchase.plan.displays}} Display License<span ng-show="purchase.plan.displays !== 1">s</span>:</span>
+      <span class="pull-right">${{purchase.estimate.subTotal}}</span>
+    </p>
+    <p class="mb-0">
+      <span>$8 per display license, per month.</span>
```
Comment (alex-deaconu) [inline web/partials/purchase/checkout-purchase-summary.html:31]: Hardcoded. Need to fix.

File: web/partials/purchase/checkout-purchase-summary.html [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+    </p>
+    <p class="mb-0">
+      <span class="font-weight-bold">Total tax:</span>
+      <span class="pull-right">${{purchase.estimate.totalTax | number:2}}</span>
+    </p>
+  </div>
+
+  <div class="py-4">
+    <p class="mb-0">
+      <span class="font-weight-bold">Total cost per month:</span>
```
Comment (alex-deaconu) [inline web/partials/purchase/checkout-purchase-summary.html:82]: Hardcoded. Will fix.

File: web/partials/purchase/checkout-purchase-summary.html [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+      </div>
+      <div class="col-xs-3">
+        <a class="btn btn-default btn-block" href="#" ng-click="factory.getEstimate()">
+          Retry
+        </a>
+      </div>
+    </div>
+  </div>
+
+  <p>
```
Comment (ezequielc) [inline web/partials/purchase/checkout-purchase-summary.html:20]: @alex-deaconu We don't have this in the mockups, right? I think it is a good addition, but maybe we should show it only when the user does not belong to the selected company. Thoughts? Thanks

**Review by ezequielc — COMMENTED** [review]
@alex-deaconu I entered a declined card (4000000000000127) to test payment failures and the error message shows 1-2 seconds after the spinner is gone. I think we need to fix that. Thanks

**Comment by ezequielc** [comment]
Also @alex-deaconu After entering Tax Exemption, I still see taxes being applied. Maybe it is just needs refreshing the estimate? Thanks!

**Review by ezequielc — APPROVED** [review]

**Pull request merged**